libsolidity/ast/Types.cpp

```diff
@@ -949,11 +949,22 @@ bool RationalNumberType::operator==(Type const& _other) const
 	return m_value == other.m_value;
 }
 
+string numberToString(dev::bigint num)
```

Comment:
This should be in the anonymous namespace or be a static member of the class.

Reply:
I also think the `dev` namespace is imported.

Reply:
The name of this should also reflect that it is truncating, perhaps a parameter for the max length would be useful?

Reply:
Yes, that's better, what is the convention for these utility functions?

Reply:
ping @axic

Reply:
There's no strict convention. For the moment I think you could keep this as a private static method of `RationalNumberType` just name it something like `bingintTruncatedToString` (or if you find something better).